service/api_impl.ml

```diff
@@ -2,6 +2,7 @@ module Rpc = Current_rpc.Impl (Current)
 module Raw = Ocaml_ci_api.Raw
 module String_map = Map.Make (String)
 module Index = Ocaml_ci.Index
+module Pipeline = Ocaml_ci.Pipeline
```

Comment:
Shouldn't this type be extracted in another module to avoid importing the pipeline? It doesn't seem that bad to have an `Experimental` module.